### PR TITLE
PA-340 Add widget init endpoint

### PR DESCRIPTION
**Pull request by alexander-rogachev-fastdev**
# Jira Issue: [PA-340]

## What?
- Add widget init endpoint

## Why?
- Need for widget initialization

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

[PA-340]: https://hedvig.atlassian.net/browse/PA-340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/generated/graphql.ts [hunk 1]
```diff
@@ -1943,6 +1943,13 @@ export type IndefinitePercentageDiscount = {
   percentageDiscount: Scalars['Float'];
 };
 
+export type InitWidgetInput = {
+  locale: Scalars['String'];
```
Comment (johantjellden) [inline src/generated/graphql.ts:1947]: This should probably be of `Locale? Line(2077) type.

File: src/resolvers/quote-cart/quoteCart.graphqls [hunk 2]
```diff
@@ -127,6 +132,13 @@ input CreateOnboardingQuoteCartInput {
     locale: String!
 }
 
+input InitWidgetInput {
+    requestId: String!
+    partnerId: String!
+    market: String!
```
Comment (johantjellden) [inline src/resolvers/quote-cart/quoteCart.graphqls:138]: Should not market should be of enum `Market`

File: src/api/upstreams/rapio.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import { factory } from '../../utils/log'
+import { HttpClient, HttpError } from '../httpClient'
+
+const logger = factory.getLogger('rapio')
+
+export interface RapioClient {
+  initWidget(
+    requestId: string,
+    partnerId: string,
+    market: string,
```
Comment (StylianosGakis) [inline src/api/upstreams/rapio.ts:12]: Should this also be of type Market?

File: src/api/upstream.ts [hunk 4]
```diff
@@ -1,5 +1,9 @@
 import * as config from '../config'
-import {ForwardHeaders, OptionalTokenProvider, TokenProvider} from '../context'
+import {
+  ForwardHeaders,
+  OptionalTokenProvider,
+  TokenProvider,
+} from '../context'
```
Comment (johantjellden) [inline src/api/upstream.ts:6]: Are we missing a prettier setup or something? Why are these lines reformatted?

Comment (alexander-rogachev-fastdev) [reply src/api/upstream.ts:6]: no idea

Comment (johantjellden) [reply src/api/upstream.ts:6]: @rasmusguterstam Do you know?

**Review by johantjellden — APPROVED** [review]

**Pull request merged**